packages/addon-dev/src/rollup.ts

```diff
@@ -64,10 +63,11 @@ export class Addon {
     return gjs(options);
   }
 
-  // By default rollup does not clear the output directory between builds. This
-  // does that.
-  clean(options: DelOptions) {
-    return clean({ targets: `${this.#destDir}/*`, ...options });
+  // this does incremental updates to the dist files and also deletes files that are not part of the generated bundle
+  // rollup already supports incremental transforms of files,
+  // this extends it to the dist files
+  clean() {
+    return clean();
```

Comment:
If we consider this breaking, we could still release this on stable as a minor version, by exposing this as a separate plugin, and updating the blueprint to use the new one instead of `addon.clean()`! 🤔